script/codesign/codesign.cnf

```diff
@@ -0,0 +1,16 @@
+[req]
+distinguished_name = req_distinguished_name
+prompt = no
+
+[req_distinguished_name]
+C = CA
+ST = BC
+L = Vancouver
+O = ElectronJS
+OU = BuildAutomation
+CN = codesign.electronjs.org
+
+[extended]
+keyUsage         = critical,digitalSignature
+extendedKeyUsage = critical,codeSigning
```

Comment:
is this extended usage actually required? the snippet you showed me didn't seem to have it

Reply:
@nornagon Yes, `security` doesn't report it as a valid code signing cert without it